src/code/z_camera.c

```diff
@@ -6778,7 +6807,8 @@ s32 Camera_Special6(Camera* camera) {
         rwData->animTimer--;
     } else {
         // Camera following player on the x axis.
-        sCameraInterfaceFlags &= 0xF0FF;
+        // Overwrite interface alpha to 50
```

Comment:
oldie comment here :older_adult: 
there's also "Overwrite interface alpha to 3" elsewhere